Quaver.Shared/Screens/Gameplay/Rulesets/Keys/HitObjects/GameplayHitObjectKeys.cs

```diff
@@ -524,13 +537,13 @@ public void FadeOut(double dt)
         ///     Starts looping the long note sprite.
         ///     It will only be initiated when the player presses the note.
         /// </summary>
-        public void StartLongNoteAnimation() => LongNoteBodySprite.StartLoop(Direction.Forward, 30);
+        //public void StartLongNoteAnimation() => LongNoteBodySprite.StartLoop(Direction.Forward, 30);
 
         /// <summary>
         ///     Stops looping the long note sprite.
         ///     It will only be initiated when the player releases the note.
         /// </summary>
-        public void StopLongNoteAnimation() => LongNoteBodySprite.StopLoop();
+        //public void StopLongNoteAnimation() => LongNoteBodySprite.StopLoop();
```

Comment:
Can't accept this if you're removing hold animation functionality. 

How about we do something like this instead:

![image](https://user-images.githubusercontent.com/20389565/161400625-d2d2336b-4088-4fe1-9ae0-86bd02604974.png)

Reply:
Fixed, column are now for the snap color, and rows are for the animation. (Also added this option for hitobjects, and even fps for the animation speed)